docs/src/components/frame/aside.module.css

```diff
@@ -0,0 +1,58 @@
+@import '@pluralsight/ps-design-system-core';
+
+@custom-media --breakpoint-small (width >= 768px);
```

Comment:
can't figure out a way to use cssvars in media queries 🤷 

also i need to figure out how to make these media definitions globally defined/available

Reply:
Yeah, I think you found the solution: https://cssnext.github.io/features/#custom-media-queries

Apparently you can't use var() in media queries: https://stackoverflow.com/a/40723269